server/src/router.ts

```diff
@@ -288,6 +298,28 @@ export class Router {
       case "whiteboard":
         // socket.broadcast.emit("event", message);
         socket.to(clientRooms[socket.id]).emit("event", message);
+        break;
+      case "weather":
+
+        console.log(message.value)
+      axios.get(`http://api.openweathermap.org/data/2.5/weather?q=${message.value}&appid=${WEATHER_APIKEY}`)
+      .then(res => {
+        let temp =res.data.main.temp ;
+        let condition = res.data.weather[0].main ;
+
+        io.emit("event", {
```

Comment:
should change to use socket.to(...).emit(...) like other code, since we want to separate events by chat rooms, io.emit sends to everyone in every room instead of `socket.to` which sends to one room only